### PR TITLE
Newsletter 6

### PR DESCRIPTION
**Pull request by jamesmunns**
Rendered: https://github.com/rust-lang-nursery/embedded-wg/blob/newsletter-6/newsletters/2018-05-28.md

### CODE REVIEW

File: newsletters/2018-05-28.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+If you want to mention something in [the next newsletter], make sure to leave a comment on the issue.
+
+[the next newsletter]: https://github.com/rust-lang-nursery/embedded-wg/issues/103
+[Embedded WG]: https://github.com/rust-lang-nursery/embedded-wg
+
+## Highlights
+
+* The [Paris edition of RustFest] has wrapped up their workshops, but the [impl-days] are taking place on the 28th and 29th of May. If you have a chance, check them out!
+* [Jacob Creedon] gave an introduction talk about using Rust with embedded systems at the [Teardown Conference] in Portland, check out [Jacob's Slides] or tye [Video of Jacob's Talk]
```
Comment (Emilgardis) [inline newsletters/2018-05-28.md]: typo
> or **tye** [Video of Jacob's Talk]

File: newsletters/2018-05-28.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+* The [Embedded WG]'s own [theJPster] talked about the Monotron at  RustFest Paris, you can see [JP's Slides], or check out the [Video if JP's Talk]
```
Comment (Emilgardis) [inline newsletters/2018-05-28.md]: typo, wrong link ref
> [Video **if** JP's Talk]

**Comment by japaric** [comment]
This will be updated to include stuff from the last few weeks (which should have been newsletter 7). Note to self:

- Add esp-rs: https://users.rust-lang.org/t/rust-on-esp8266/12933/8

**Comment by japaric** [comment]
> Note to self:

Also: collect commercial testimonials: https://github.com/rust-lang-nursery/embedded-wg/issues/108

**Comment by japaric** [comment]
Updated with stuff from the past few weeks. r? @jamesmunns 

@jamesmunns If everything looks good to you and nothing else seems missing could you make the release (internals forum + reddit + twitter)? (I don't have access to the embedded Rust twitter account).

**Comment by hannobraun** [comment]
Seems to be missing the lpc82x-hal 0.2 release: https://github.com/rust-lang-nursery/embedded-wg/issues/103#issuecomment-398512252

**Comment by jamesmunns** [comment]
@japaric I'll add the item from @hannobraun, and publish. I should hopefully get to this today or tomorrow (and then get back on the 2 week regular plan).

Thanks for the ping!

**Comment by jamesmunns** [comment]
Rebased, will merge now.

**Pull request merged**